conf/docker/solr/entrypoint.sh

```diff
@@ -2,17 +2,36 @@
 
 # FIXME: Don't run Solr out of /tmp!
```

Comment:
You fixed it so that Solr doesn't run out of "/tmp" anymore, right? Thanks! If so, can you please remove this comment?